docs/topics/sealed-classes.md

```diff
@@ -43,20 +24,14 @@ data class Sum(val e1: Expr, val e2: Expr) : MathExpr()
 object NotANumber : Expr
 ```
 
-## Location of direct subclasses
-
-All direct subclasses of a sealed class must be declared in the same file as this class itself. Classes that extend
-direct subclasses of a sealed class (indirect inheritors) can be placed anywhere, not necessarily in the same file.
+A sealed class is [abstract](classes.md#abstract-classes) by itself, it cannot be instantiated directly and can have `abstract` members.
 
-### Additional location: the same package
+Sealed classes are not allowed to have non-`private` constructors (their constructors are `private` by default).
```

Comment:
Actually, constructors of `sealed` classes are `protected` by default,  and it's allowed to use `private` or `protected` visibility on them. `public` and `internal` are prohibited

Reply:
```kotlin
sealed class Base() {
    private constructor(x: Int) : this()
    protected constructor(x: Double) : this()
    <!NON_PRIVATE_OR_PROTECTED_CONSTRUCTOR_IN_SEALED!>internal<!> constructor(x: String) : this()
    <!NON_PRIVATE_OR_PROTECTED_CONSTRUCTOR_IN_SEALED!>public<!> constructor(x: Char) : this()

    class A0 : Base()
    class A1 : Base(1)
    class A2 : Base(1.0)
    class A3 : Base("")
    class A4 : Base('a')
}
```

Reply:
Thanks. Corrected the text and added an example.